services/src/test/scala/cromwell/services/metadata/impl/MetadataDatabaseAccessSpec.scala

```diff
@@ -262,6 +262,36 @@ class MetadataDatabaseAccessSpec extends FlatSpec with Matchers with ScalaFuture
           val resultByName = response.results groupBy (_.name)
           resultByName.keys.toSet.flatten should equal(Set(Workflow1Name, Workflow2Name))
         }
+        // Filter by exclude label using AND
+        _ <- dataAccess.queryWorkflowSummaries(WorkflowQueryParameters(Seq(
+          WorkflowQueryKey.ExcludeLabelAndKeyValue.name -> s"${testLabel2.key}:${testLabel2.value}"))) map { case (response, _) =>
+          val resultByName = response.results groupBy (_.name)
+          resultByName.keys.toSet.flatten should equal(Set(Workflow1Name))
```

Comment:
Could we have a test which specifies two labels so that we get a different answer for the AND and the OR cases?

Reply:
I do have that test, but surprisingly they both return workflows with same `WorkflowName = Workflow1Name (= 'test1')` but different number of `totalResultsCount`. So I am adding that condition check as well inside the same tests.

Reply:
oh, right, because the same workflow can be submitted with different names? Sounds good to me

Reply:
Awesome! Thanks! :)